frontend/model/contracts/group.js

```diff
@@ -120,6 +120,15 @@ function memberLeaves ({ username, dateLeft }, { meta, state, getters }) {
   updateCurrentDistribution({ meta, state, getters })
 }
 
+function isContractYoungerThanUser (contractMetaData, userProfile) {
+  if (!contractMetaData || !userProfile) { return }
+
+  const contractCreated = new Date(contractMetaData.createdDate)
+  const userJoined = new Date(userProfile.joinedDate)
+
+  return contractCreated > userJoined
+}
+
```

Comment:
I don't understand the purpose of this function. It seems unnecessary.

Reply:
Oh, I missed your comment here: https://github.com/okTurtles/group-income/pull/1369#discussion_r933960575

FYI, these sorts of comments would be much better placed inline in the code than on Github. I will re-evaluate this function in light of that comment, but if you could also add that comment in the codebase it would be appreciated!